examples/ember-cli/todomvc/app/components/todo-item.js

```diff
@@ -0,0 +1,44 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+	repo: Ember.inject.service(),
+	tagName: 'li',
+	editing: false,
+	classNameBindings: ['completed', 'editing'],
+
+	actions: {
+		startEditing() {
+			this.attrs.onStartEdit();
```

Comment:
definitely *not*. `this.get('onStartEdit')();` seems good, or `this.sendAction('onStartEdit');`.